charts/gardener-extension-shoot-cert-service/templates/deployment.yaml

```
@@ -33,6 +33,8 @@ spec:
         app.kubernetes.io/instance: {{ .Release.Name }}
         networking.gardener.cloud/to-runtime-apiserver: allowed
         networking.gardener.cloud/to-dns: allowed
+        networking.gardener.cloud/to-public-networks: allowed
+        networking.gardener.cloud/to-private-networks: allowed
```

```suggestion
        networking.gardener.cloud/to-dns: allowed
        networking.gardener.cloud/to-apiserver: allowed
```

If it's only about accessing the API server, this should do the trick as well.